cli/init.js

```diff
@@ -1,32 +1,28 @@
 import fs from 'fs'
 import path from 'path'
 
-import { MOULD_DIRECTORY, RESOLVERS_JS, RESOLVERS_TS } from './constants'
+import * as paths from './paths'
 
-const originalDirectory = process.cwd()
-
-const tsconfigPath = path.join(originalDirectory, 'tsconfig.json')
-const mouldPath = path.join(originalDirectory, MOULD_DIRECTORY)
-
-const useTs = fs.existsSync(tsconfigPath)
-
-const resolversPath = path.join(mouldPath, useTs ? RESOLVERS_TS : RESOLVERS_JS)
```

Comment:
Moved this logic over to [`paths.js`](https://github.com/mouldjs/mould/pull/89/files#diff-32c07bd709660b7cd72b50de39fa7bfcR11)